src/main/java/com/icthh/xm/tmf/ms/document/service/generation/DocumentRendererType.java

```diff
@@ -14,7 +14,7 @@
 @RequiredArgsConstructor
 @Getter
 public enum DocumentRendererType {
-    JASPER_REPORTS(of(APPLICATION_PDF, TEXT_XML, APPLICATION_XML, APPLICATION_OCTET_STREAM));
+    JASPER_REPORTS(of(APPLICATION_PDF, TEXT_XML, APPLICATION_XML, APPLICATION_OCTET_STREAM, MediaType.valueOf("application/vnd.openxmlformats-officedocument.wordprocessingml.document")));
```

Comment:
Could you please extract this to some constant and re-use in `src/main/java/com/icthh/xm/tmf/ms/document/helper/DocxDocumentHelper.java` ?

Reply:
done